decidim-forms/app/forms/decidim/forms/answer_form.rb

```diff
@@ -18,7 +18,7 @@ class AnswerForm < Decidim::Form
       validates :selected_choices, presence: true, if: :mandatory_choices?
 
       validate :max_choices, if: -> { question.max_choices }
-      validate :all_choices, if: -> { question.question_type == "sorting" }
+      validate :all_choices, if: -> { question.question_type == "sorting" && question.mandatory }
```

Comment:
This seems like a functionality improvement, that needs to be ported in the develop branch as well. 

Please Refactor this to something like : 

```
validate :all_choices, if: -> { sorting? && question.mandatory? }
```
make sure that `mandatory` field is a boolean, and mark it accordingly (In rails / ruby we use ? as suffix if there is a boolean)

After you make the changes, I will do the backport to 0.27.